test/unit/unit_test.html

```diff
@@ -20,6 +20,7 @@
   <script src="crypto_spec.js"></script>
   <script src="evaluator_spec.js"></script>
   <script src="stream_spec.js"></script>
+  <script src="chunkedstream_spec.js"></script>
```

Comment:
Nit: ideally this file should be named `chunked_stream_spec.js`, to match the name of the file being tested.